grails-app/views/project/_details.gsp

```diff
@@ -1,434 +1,506 @@
+                                   id="plannedEndDate" data-validation-engine="validate[future[plannedStartDate]]"/>
+                    <g:message code="project.details.plannedEndDate.extra"/>
+                </div>
+            </div>
+
+            <div id="associatedOrgs">
+                <div class="row-fluid">
+                    <div class="clearfix control-group">
+                        <label class="control-label span3" for="associatedOrgList"><g:message code="project.details.associatedOrgs"/>:<fc:iconHelp><g:message code="project.details.associatedOrgs.help"/></fc:iconHelp></label>
+                        <div class="span6" id="associatedOrgList">
+                            <!-- ko foreach: associatedOrgs -->
+                            <div class="span12 margin-left-0 margin-bottom-1">
+                                <div class="span5 margin-left-0" data-bind="text: name"></div>
+                                <div class="span5"><img src="" data-bind="visible: logo, attr: {src: logo}" alt="Organisation logo" class="small-logo"></div>
+                                <div class="span2"><a href="#" data-bind="click: $parent.removeAssociatedOrganisation, attr: {'data-value': id}"><i class="fa fa-remove">&nbsp;</i><g:message code="project.details.associatedOrgs.remove"/></a></div>
+                            </div>
+                            <!-- /ko -->
+                        </div>
+                    </div>
+                </div>
+                <div data-bind="with: associatedOrganisationSearch">
+                    <div id="addAssociatedOrgPanel" class="span12">
+                        <div class="row-fluid">
+                            <div class="span3"></div>
+                            <div class="span9">
+                                <div class="clearfix control-group">
+                                    <label class="control-label left-aligned-label span3" for="associatedOrgName"><g:message code="project.details.associatedOrgs.name"/><i class="req-field"></i></label>
+
+                                    <div class="controls span12 margin-left-0">
+                                        <input id="associatedOrgName" class="input-xxlarge" type="text" placeholder="Start typing a name here" data-bind="value:term, valueUpdate:'afterkeydown'"><button class="btn" type="button" data-bind="click:clearSelection"><i class='icon-search' data-bind="css:{'icon-search':!term(), 'icon-remove':term()}"></i></button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="organisation-search">
+                            <div class="row-fluid">
+                                <div class="span3"></div>
+                                <div class="span8">
+                                    <div class="control-label span12">
```

Comment:
Yep, I wanted the checkbox to be right aligned on the page. I have changed this to add a ```pull-right``` class on the label rather than using control-label on the div since it's a bit more obvious.